docs/setup/local-up.md

```diff
@@ -2,7 +2,7 @@
 
 Once you have docker running, you can create a local Sedna cluster with:
 ```
-bash hack/local-up.sh
+bash scripts/local-up.sh

```

Comment:
This puzzles me because I didn't find this `shell script` in the `scripts` directory.

Reply:
updated